cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -2214,6 +2214,110 @@
           "additionalProperties": false
         }
       }
+    },
+    "reporting_config": {
+      "type": "object",
+      "properties": {
+        "reporting": {
+          "type": "object",
+          "additionalProperties": false,
```

Comment:
Stylistic comment (no changes requested): I like the way this is structured. Our schema has a lot of individual statements tacked on to the end of long {} contexts, which I personally I find slightly harder to visually parse:
```

        }
      },
      "additionalProperties": true
    }
  ]
}
```

vs:
```json
       "additionalProperties": true,
       "properties": {

```
Going forward I would prefer to the second style where possible.